ci/Dockerfile.python3.6

```diff
@@ -0,0 +1,199 @@
+# Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
There's a lot of duplicity with Dockerfile.python2.7 - can't we use multi stage builds to eliminate duplicity by using a common base Dockerfile?

Reply:
We don't update these image often. Merge them won't save us time. And make it more effort to follow aws-codebuild-docker-images updates.